src/test/java/com/sweng894/GetVaccinated/PersonTests.java

```diff
@@ -0,0 +1,36 @@
+package com.sweng894.GetVaccinated;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class PersonTests {
+
+	private Person person;
+
+	/**
+	 * @throws java.lang.Exception
+	 */
+	@BeforeEach
+	void setUp() throws Exception {
+		person = new Person("John Doe", "2", "N");
+	}
+
+	@Test
+	void testEligiblePerson() {
+
+		// Person has not been vaccinated and is not in the current rollout phase
+		if (("1B".compareTo(person.getPhase())) <= 0) {
+			assertTrue("N".equals(person.getIsVaccinated()), person.getName() + " is ineligible.");
+		}
```

Comment:
Having conditions in the test code can make it more difficult to tell what is being tested. We probably should have the logic inside the class under test, and have test cases that ensure the branches are covered.

Reply:
What do you mean by having the log "inside the class under test"? Have the logic in the Person.java file?

Reply:
Yes, the if statement would be somewhere in the Person.java file.